source/development_process/index.rst

```diff
@@ -0,0 +1,41 @@
+The OSL Development Workflow
+============================
+
+The OSL builds websites for a variety of clients including the Oregon Virtual
+School Districts, the Oregon Sea Grant Foundation, the OSU Biology department
+and Google. When the initial development is finished, the OSL will host and
+maintain the websites. This document will guide developers through the steps of
+creating a new website for a client. New developers are encouraged to read this
+document to understand how the developer team works, and experienced developers
+are encouraged to review this document to brush up their knowledge of parts of
+the process they may not be intimately familiar with. This workflow description
+will also be available to clients so they can understand how the OSL developer
+team works.
+
+.. toctree::
+  :maxdepth: 1
+  :glob:
+
+  *
+
+Tools The Team Uses
+-------------------
+
+
+At the OSL developers use a variety of open source tools. Developers are
+expected to have at least passing familiarity with all of these tools. New
+developers are encouraged to research any of these tools they have not heard
+of.
```

Comment:
I am of the opinion that that is a perfectly good phrase to end a sentence with. Seriously, that's not an issue in the least, prescriptivists notwithstanding. They're the only ones who ever bring it up. But their ideology is going down!

(But seriously, they're fine, they're English.)